### PR TITLE
Hotfix/resource load

### PR DESCRIPTION
**Pull request by Segelzwerg**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -9,3 +9,18 @@ Assets/Music/510953__theojt__cinematic-battle-song.mp3 filter=lfs diff=lfs merge
 Assets/Music/510953__theojt__cinematic-battle-song.mp3.meta filter=lfs diff=lfs merge=lfs -text
 Assets/Music/A1-Cave.mp3 filter=lfs diff=lfs merge=lfs -text
 Assets/Music/A1-Cave.mp3.meta filter=lfs diff=lfs merge=lfs -text
```
Comment (iTitus) [inline .gitattributes]: is this still necessary?

File: .gitattributes [hunk 2]
```diff
@@ -9,3 +9,18 @@ Assets/Music/510953__theojt__cinematic-battle-song.mp3 filter=lfs diff=lfs merge
...
+Assets/Resources/Music/510953__theojt__cinematic-battle-song.mp3 filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Music/510953__theojt__cinematic-battle-song.mp3.meta filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Music/A1-Cave.mp3 filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Music/A1-Cave.mp3.meta filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Sounds/Ambient filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Sounds/Ambient.meta filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Sounds/Foleys filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Sounds/Foleys.meta filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Sounds/Mixer filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Sounds/Mixer.meta filter=lfs diff=lfs merge=lfs -text
```
Comment (iTitus) [inline .gitattributes]: can you use wildcards ( ** ) here?

File: doc/developer_guide/developer_guide.adoc [hunk 3]
```diff
@@ -14,4 +14,6 @@ the `index.adoc` of the directory.
 
 === Tutorials
 
-1. link:./Sound/index.adoc[Sound Tutorials]
\ No newline at end of file
+1. link:./Sound/index.adoc[Sound Tutorials]
+
+2. link:../Resources/index.adoc[Resource Tutorialsw]
```
Comment (iTitus) [inline doc/developer_guide/developer_guide.adoc]:
```suggestion
2. link:../Resources/index.adoc[Resource Tutorials]
```

**Review by iTitus — CHANGES_REQUESTED** [review]

File: .gitattributes (hunk 2 above)
Comment (Segelzwerg) [reply .gitattributes]: I don't know, I used the official LFS tutorial using the Terminal and never touched the file manually.

File: .gitattributes (hunk 1 above)
Comment (Segelzwerg) [reply .gitattributes]: Yeah mayb, as I said below, never wrote in the file manually.

File: .gitattributes (hunk 2 above)
Comment (Thabb) [reply .gitattributes]: It works exactly like .gitignore
You can track all files in a folder with 
`{path}/** filter=lfs diff=lfs merge=lfs -text`
Or all files of a certain type with 
`{path}/*.[fileending] filter=lfs diff=lfs merge=lfs -text`
Would probably be a better way than adding them all one by one

Comment (Thabb) [reply .gitattributes]: You can also use 
`git lfs track {path}/**`

Comment (Segelzwerg) [reply .gitattributes]: will do!

**Review by iTitus — APPROVED** [review]

**Pull request merged**